quantum/usb_descriptor_common.h

```diff
@@ -0,0 +1,31 @@
+/* Copyright 2020 Nick Brassel (tzarc)
```

Comment:
Any chance we move this to `tmk_core/protocol/usb_descriptor_common.h`?

Reply:
Done.